src/main.js

```diff
@@ -80,9 +80,119 @@ function createWindow() {
             },
             actions.inspect(),
             actions.services(),
+            actions.separator(),
+			{
+				label: "Change Language",
+				submenu: [
+					{
+						label: "Bulgarian",
+						click: () => {
+							reloadWithLanguage("bg");
+						}
+					},
+					{
+						label: "Czech",
+						click: () => {
+							reloadWithLanguage("cs");
+						}
+					},
+					{
+						label: "German",
+						click: () => {
+							reloadWithLanguage("de");
+						}
+					},
+					{
+						label: "Greek",
+						click: () => {
+							reloadWithLanguage("el");
+						}
+					},
+					{
+						label: "English",
+						click: () => {
+							reloadWithLanguage("en");
+						}
+					},
+					{
+						label: "Spanish",
+						click: () => {
+							reloadWithLanguage("es");
+						}
+					},
+					{
+						label: "French",
+						click: () => {
+							reloadWithLanguage("fr");
+						}
+					},
+					{
+						label: "Hungarian",
+						click: () => {
+							reloadWithLanguage("hu");
+						}
+					},
+					{
+						label: "Italian",
+						click: () => {
+							reloadWithLanguage("it");
+						}
+					},
+					{
+						label: "Lithuanian",
+						click: () => {
+							reloadWithLanguage("lt");
+						}
+					},
+					{
+						label: "Polish",
+						click: () => {
+							reloadWithLanguage("pl");
+						}
+					},
+					{
+						label: "Portuguese",
+						click: () => {
+							reloadWithLanguage("pt");
+						}
+					},
+					{
+						label: "Romanian",
+						click: () => {
+							reloadWithLanguage("ro");
+						}
+					},
+					{
+						label: "Russian",
+						click: () => {
+							reloadWithLanguage("ru");
+						}
+					},
+					{
+						label: "Swedish",
+						click: () => {
+							reloadWithLanguage("sv");
+						}
+					},
+					{
+						label: "Turkish",
+						click: () => {
+							reloadWithLanguage("tr");
+						}
+					},
+					{
+						label: "Ukrainian",
+						click: () => {
+							reloadWithLanguage("uk");
+						}
+					},
+					// Add more languages as needed
```

Comment:
This definitely can be simplified, also you can use language list from here [https://stackoverflow.com/a/69034315](url)

Reply:
agreed that would simplify it